feature-libs/user/profile/components/register/register.component.html

```diff
@@ -184,6 +184,11 @@
               </label>
             </div>
           </div>
+          <cx-captcha
+            [control]="registerForm.get('captcha')"
+            (enabled)="captchaEnabled($event)"
```

Comment:
I think we could manipulate the form control from within the captcha component so that we do not need to have extra logic to handle `true` or `false` in the registration component.